CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Fixed
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
+* Fixed `realm_refresh` so it uses an argument value for the refresh result and returns any error conditions as return value. ([#6068](https://github.com/realm/realm-core/pull/6068))
```

Comment:
Probably just one line is needed, stating that we changed both functions.

Reply:
Kept it separate since these are not related fixes